decidim-meetings/lib/decidim/meetings/feature.rb

```diff
@@ -23,6 +23,8 @@
     meetings.count
   end
 
+  feature.actions = %w(join)
```

Comment:
This allows setting permissions per action. For example, preventing a user to register to a meeting unless it's validated or from a certain postal code.